BUILD

```diff
@@ -0,0 +1,6 @@
+# Google's Cross-Publisher Reach Experiments
```

Comment:
I don't think we want to mention Google in this.

Reply:
Let's also wait for formal confirmation Bazel is ok to use.

Reply:
I will definitely remove this!  Our internal tooling does not sync README files by default so oddly enough our internal readme does not mention google.  I will dig into this and figure out the best way forward, and push a new commit to this branch shortly.  Thank you!

Reply:
just realized this is a BUILD file actually, will remove it from this commit and do a more earnest BUILD migration later if we do use bazel.  stand by